udata/tests/test_discussions.py

```diff
@@ -100,6 +100,44 @@ def test_new_discussion_missing_subject(self):
         })
         self.assertStatus(response, 400)
 
+    def test_new_discussion(self):
```

Comment:
The function must be renamed (`test_new_discussion_with_extras` ?) because it's already used (and so one of the test is hiding the other)